frontend/src/utils/apiConstants.js

```diff
@@ -0,0 +1,16 @@
+export const getUrl = () => {
+  const { hostname, protocol } = window.location;
+  return hostname === "localhost"
+    ? "http://localhost:8000"
+    : `${protocol}//${hostname}`;
+};
+
+export const BASE_URL = getUrl();
+
+export const OCP_JOBS_API_V1 = "/api/v1/ocp/jobs";
```

Comment:
I am not sure where these URLs are being used in the code.

Reply:
As discussed, these URLs will be used in the future.